stack/dashboard/base/files/etc/dashboard.yml

```diff
@@ -0,0 +1,16 @@
+server.host: 0.0.0.0
+server.port: 443
+opensearch.hosts: https://localhost:9700
+opensearch.ssl.verificationMode: certificate
+#opensearch.username: 
```

Comment:
Why commented?

Reply:
fc6b6306

Reply:
I only wanted to understand why it was commented because I had no notice about the keystore. I prefer it commented.